libqtile/widget/moc.py

```diff
@@ -36,11 +36,12 @@ class Moc(base.ThreadPoolText):
     defaults = [
         ('play_color', '00ff00', 'Text colour when playing.'),
         ('noplay_color', 'cecece', 'Text colour when not playing.'),
-        ('max_chars', 0, 'Maximum number of characters to display in widget.')
+        ('max_chars', 0, 'Maximum number of characters to display in widget.'),
+        ('update_interval', 0.5, 'Update Time in seconds.'),
```

Comment:
Once you've made the changes I suggested, this should then be the only change you need for this widget